<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>304</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExampleUIFile</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="uiCubeName">
         <property name="text">
          <string>cube</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="uiCubeLength">
         <property name="value">
          <double>8.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="uiCreateCube">
       <property name="text">
        <string>Create Cube</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
